src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -60,6 +61,18 @@ public function load(mixed $resource, ?string $type = null): mixed
         // Force load ContainerConfigurator to make env(), param() etc available.
         class_exists(ContainerConfigurator::class);
 
+        $autoloaderRegistered = false;
+        if (!$this->configBuilderAutoloader) {
+            $this->configBuilderAutoloader = function (string $class) {
+                if ($this->generator && str_starts_with($class, 'Symfony\\Config\\') && str_ends_with($class, 'Config')) {
```

Comment:
This condition can move out of the closure.
```suggestion
        if (!$this->configBuilderAutoloader && $this->generator) {
            $this->configBuilderAutoloader = function (string $class) {
                if (str_starts_with($class, 'Symfony\\Config\\') && str_ends_with($class, 'Config')) {
```

Reply:
Indeed!